### PR TITLE
gp.sh uses http instead of git protocol

### PR DESCRIPTION
**Pull request by cletusw**
Don't know if this affects anyone else, but the git protocol is sometimes blocked by firewalls for me.

### CODE REVIEW

**Comment by ssorallen** [comment]
I'm in favor of HTTPS instead of SSH as well because GitHub suggests it. I opened a similar issue for the pull-all script: https://github.com/Polymer/tools/issues/44

File: bin/gp.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ branch=${3:-"master"} # default to master when branch isn't specified
 
 # make folder (same as input, no checking!)
 mkdir $repo
-git clone git@github.com:$org/$repo.git --single-branch
+git clone http://github.com/$org/$repo.git --single-branch
```
Comment (ssorallen) [inline bin/gp.sh]: This should probably be "https". GitHub redirects to https anyway, so that will save the script a redirect.

Comment (cletusw) [inline bin/gp.sh]: Done :)

**Comment by rictic** [comment]
Outdated, from a previous use of this repo.

**Pull request closed without merging**